hosted-workshop-prod: Update to v3.0.5

joshix opened the pull request:
Copy the latest release of hosted-workshop-production.json from
https://github.com/openshift-labs/workshop-spawner/blob/3.0.5/templates/hosted-workshop-production.json

joshix committed:
hosted-workshop-prod: Update to v3.0.5

Copy the latest release of hosted-workshop-production.json from
https://github.com/openshift-labs/workshop-spawner/blob/3.0.5/templates/hosted-workshop-production.json

GrahamDumpleton commented:
Does something elsewhere override ``APPLICATION_NAME`` template parameter when it is instantiated. I recollect the instance needing to be called ``terminal`` or ``terminals`` to matched documentation and links elsewhere. The prior version hadn't changed default value, so presuming it is overridden elsewhere.

joshix commented:
Yes, in https://github.com/openshift-labs/starter-guides/blob/ab79bc255ab22b8743899c90c0031776cf54e668/apb/roles/provision-java-starter-guides/tasks/install-terminals.yml#L22

joshix committed:
templates/etherpad.txt.j2: Fix json escaping bug

Brute force: After https://github.com/openshift-labs/starter-guides/pull/115,
the etherpad pod fails during provisioning, logging the error:

"Started Etherpad... [31m[2019-04-08 01:19:31.384] [ERROR] console -
[39mThere was an error processing your settings.json file: Unexpected
token ' in JSON at position 876"

Rather than fiddle with escapes, this change eliminates both the
apostrophes.
